app/Models/Book.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Models;
+
+use App\Models\Like as ModelsLikes;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Books extends Model
+{
+    use HasFactory;
+
+    protected $table = 'books';
+    protected $primaryKey = 'id';
+
+    public function author()
+    {
+        return $this->belongsTo(Author::class);
+    }
+    public function publisher()
+    {
+        return $this->belongsTo(Publisher::class);
+    }
+    public function review()
```

Comment:
```suggestion
    public function reviews()
```
Dùng để gọi hasMany thì dùng có s em nhé.
Tương tự fix hết nha